src/main/java/seedu/duke/assets/Doctor.java

```diff
@@ -22,14 +22,13 @@ public String saveString() {
 
     @Override
     public String toString() {
-        return "Doctor {"
-                + "Nric='" + getNric() + '\''
+        return  "Nric='" + getNric() + '\''
                 + ", Full Name='" + getFullName() + '\''
                 + ", Age=" + getAge()
                 + ", Gender=" + getGender()
                 + ", Address='" + getAddress()
                 + '\'' + ", DOB='" + getDob() + '\''
-                + ", Specialization='" + getSpecialization() + '\'' + '}';
+                + ", Specialization='" + getSpecialization() + '\'';
```

Comment:
Do we need to separate this into new lines '\n'? 
Cos currently we need to have a new line in order for printParagraph() to work
And toString does not have any new lines in it, right?
So that the output print would not exceed 120 characters?

Reply:
this is supposed to be implemented by the table function in the future, so that everything is visible.

Reply:
I see I see